packages/integration-tests/sfnStep.js

```diff
@@ -23,6 +24,7 @@ class SfnStep {
    * @returns {Object} - Parsed step input object
    */
   static async parseStepMessage(stepMessage, stepName) {
+    deprecate('@cumulus/integration-tests/sfnStep/SfnStep.parseStepMessage()', '1.20.0', '@cumulus/message/StepFunctions.parseStepMessage()');
```

Comment:
This is kinda complicated, but here goes:

- I previously copied this file from common/sfnStep to here, then deprecated common/sfnStep
- I went to delete common/sfnStep in this PR but couldn't immediately do so because it was still used by packages/api/lib/cwSfExecutionEventUtils.js. And I didn't want the api package to depend on the integration-tests package, because that makes no sense
- So I created message/StepFunctions/parseStepMessage and
  - updated integration-tests/sfnStep.parseStepMessage to use it (this change)
  - updated  packages/api/lib/cwSfExecutionEventUtils.js to use it: https://github.com/nasa/cumulus/pull/1547/files?file-filters%5B%5D=.js&hide-deleted-files=true#diff-96bec6e7209fc87db891b7bceaa35c80R71
- Thus I can delete common/sfnStep, which this PR also does